src/Acuminator/Acuminator.Tests/Tests/StaticAnalysis/DacKeyFieldDeclaration/Sources/IdentityKey+Key_RemoveIdentityAttr_Expected - Copy.cs

```diff
@@ -0,0 +1,27 @@
+﻿using PX.Data;
```

Comment:
@saloponov forgot to change filename: IdentityKey+Key_RemoveIdentityAttr_Expected - Copy.cs is a remnant from copy paste